lib/common/utils.js

```diff
@@ -93,6 +99,8 @@ export const getErrorParameters = function (args) {
  * @returns {boolean} True if the object has set any data which is not `null`, `undefined` or an empty string.
  */
 export const objectHasData = function (obj) {
+  if (!obj) return false;
```

Comment:
When is this function not passed an object?

Reply:
When there is no `data` argument passed to the event method.